fenix/docs/Guide-to-merging-contributor-PRs.md

```diff
@@ -1,52 +1,22 @@
 # Guide to merging contributor PRs for Firefox Android team members
```

Comment:
Perhaps also add instructions on how to edit and push changes for contributor PR as well?

Reply:
Sure, I'll do this as a follow up.  It'll be more complicated.